src/smc-webapp/project_actions.ts

```diff
@@ -812,9 +817,7 @@ export class ProjectActions extends Actions<ProjectStoreState> {
     // sagenb worksheet (or backup of it created during unzip of multiple worksheets with same name)
     alert_message({
       type: "info",
-      message: `Opening converted CoCalc worksheet file instead of '${
-        opts.path
-      }...`
+      message: `Opening converted CoCalc worksheet file instead of '${opts.path}...`
```

Comment:
This has a typo (missing close quote), though that's not introduced in this patch.